templates/security/livepatch.html

```diff
@@ -9,7 +9,7 @@
 meta_copydoc %}
 
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom is-bordered">
+<section class="p-strip u-no-padding--top is-bordered">
   <div class="p-strip--suru-topped">
     <div class="row u-equal-height">
       <div class="col-7">
```

Comment:
Should we hide the video on smaller screens or is it something we want to keep visible?

Reply:
We should keep it visible IMO